client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -375,6 +451,10 @@ class EmailProvidersComparison extends Component {
 			return null;
 		}
 
+		const isEligibleForFreeTrial =
+			config.isEnabled( 'emails/google-workspace-1-month-trial' ) &&
+			( hasCartDomain || isDomainEligibleForGoogleWorkspaceFreeTrial( domain ) );
```

Comment:
Since we only show this old `Email Comparison` page in the domain purchase flow, `domain` will always be `undefined`. That means we can actually remove this call to `isDomainEligibleForGoogleWorkspaceFreeTrial()` as it will always return `false` (yes, we didn't need that selector yet 😀):

```suggestion
		const isEligibleForFreeTrial = config.isEnabled( 'emails/google-workspace-1-month-trial' ) && hasCartDomain;
```